Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -86,10 +87,39 @@ void HTMLScriptElement::didFinishInsertingNode()
     ScriptElement::didFinishInsertingNode();
 }
 
-// https://html.spec.whatwg.org/multipage/scripting.html#dom-script-text
 void HTMLScriptElement::setText(String&& value)
 {
-    setTextContent(WTFMove(value));
+    setText(std::variant<RefPtr<TrustedScript>, String>(WTFMove(value)));
```

Comment:
Shouldn’t this call `setTextContent` instead? We don’t need the trusted check here.

Reply:
Ah yeah good point